cfg/config.go

```diff
@@ -670,7 +670,7 @@ func BuildFlagSet(flagSet *pflag.FlagSet) error {
 		return err
 	}
 
-	flagSet.StringP("custom-endpoint", "", "", "Specifies an alternative custom endpoint for fetching data. The custom endpoint must support the equivalent resources and operations as the GCS JSON endpoint, https://storage.googleapis.com/storage/v1. If a custom endpoint is not specified, GCSFuse uses the global GCS JSON API endpoint, https://storage.googleapis.com/storage/v1.")
+	flagSet.StringP("custom-endpoint", "", "", "Specifies an alternative custom endpoint for fetching data.  The custom endpoint must support resources equivalent to storage.googleapis.com:443.  The port number is required for the gRPC-based control client used at mount time to determine the bucket type (HNS or Flat).  If a custom endpoint is not specified, GCSFuse uses the default endpoint, which is storage.googleapis.com:443. Ref: https://pkg.go.dev/google.golang.org/api/option#WithEndpoint")
```

Comment:
How about this?

To specify a custom storage endpoint, ensure it supports the same resources as the default `storage.googleapis.com:443` and includes the port number.

I think, just explicitly saying to include port number should be sufficient.